gh2/csv.py

```diff
@@ -207,24 +209,26 @@ def set_headers(repo, labels=None):
     return headers
 
 
-def main():
-    parser = make_parser()
+def get_all_labels(repositories):
```

Comment:
@git-harry yeah, it's currently broken. I'll need to fix this later, either by modifying ```issue_to_dict``` to work with a passed in list of label names instead of objects, or working to extract the names within the function for comparison.